### ISSUE TITLE
Crashes on opening scad file

### ISSUE THREAD

**Issue by salamanders** [openscad/openscad#1783]
OpenSCAD version 2016.08.21 (git c555723) 
Crashes on opening either .scad file in http://www.thingiverse.com/thing:82620
OSX 10.11.6

Release version OpenSCAD version 2015.03-3 does not crash.

**Comment by t-paul** [comment]
Both work fine with OpenSCAD version 2016.08.25.nightly (git 733562b) on Linux.

**Comment by salamanders** [comment]
I'm also getting a crash with the following hull command.  Possibly related?  Does it work in your nightly?
```
$fn =100;

trac_radius=100;
trac_width=20;
trac_thickness=3;
// Also = number of teeth!  
// http://www.robertcailliau.eu/Alphabetical/L/Lego/Gears/Rolling%20Diameters
num_teeth = trac_radius*2;
tooth_base_width    = 2.3;
tooth_tip_width     = 0.87;
brick_width         = 7.96;
tooth_height        = 2.06;

hull() {
translate([0,tooth_height,0])
cube([tooth_tip_width, 0.1,brick_width], center=true);
cube([tooth_base_width, 0.1,brick_width], center=true);
}

**Comment by t-paul** [comment]
Yes, also no issue with that `hull()` on Linux. This might be some OSX specific issue, I don't see any relevant changes between 2016-08-21 and 2016-08-25.

**Comment by salamanders** [comment]
Drat. Any part of the crash log that might help? Debug mode I could run?

**Comment by kintel** [comment]
@salamanders I've tested on 10.10.5 and 10.11.5 without managing to reproduce the crash.
A crash log would be helpful.
As would the output of Help->Library Info

**Comment by salamanders** [comment]
Crash log and Help-Library 
https://gist.github.com/salamanders/a9c05e0a0c33b9c9f5ad1af809c52ce4

**Comment by kintel** [comment]
This looks like this same issue as this thread from the mailing list: http://forum.openscad.org/Cylinder-in-Hull-crashes-2016-06-25-Mac-OS-10-9-5-td17917.html

Apparently it was solved by building from source, so it could be a build system issue on my side.
I'd still like to reproduce this so I can figure out what went wrong and potentially detect that in the future.

**Comment by salamanders** [comment]
I too solved it by building from source.   (Once I got around #1784 )

**Comment by kintel** [comment]
Latest reports indicate this:
* Binaries built using Xcode 7 (on Mac OS X 10.10) are broken
* Binaries built using Xcode 8 (on Mac OS 10.11- that's the minimum required OS version) works
* OpenSCAD _executable_ built on Xcode 7 using _libraries_ built on Xcode 8 works, but _not_ vice versa

-> one or more of the libraries are broken on the Xcode7 build.

**Comment by kintel** [comment]
Fixed by #1858 
OpenSCAD-2016.11.06 has this fix.
